tests/utils.go

```diff
@@ -4533,6 +4548,7 @@ func resetToDefaultConfig() {
 		return
 	}
 
+	RestoreToE2eKubevirtResource()
 	UpdateKubeVirtConfigValueAndWait(KubeVirtDefaultConfig)
```

Comment:
This looks to me like we should have two ways of setting something on the KubeVirt CR:

 1. with `UpdateKubeVirtConfigValueAndWait(config)` which people would use in their tests
 2. the general cleanup in this function after each tests. `UpdateKubeVirtCRAndWait(e2ecr)`
 
 It may not be an issue or not, but just looking at this code I am not sure if we would wait int all cases that the changes got received by all components. It may for instance be the case that `RestoreToE2eKubevirtResource()` would revert the CR and `UpdateKubeVirtConfigValueAndWait` would think that the CR is already in sync and it would not check all components.

To sum up, it may make sense to collaps the two calls here to one function which reverts the whole resource and waits for  it to be propagated.

One additional note: I would have expected that only the operator tests are changing the KV resource outside of the configuration section. Let's just ensure that the revert stays efficient.

Reply:
@rmohr Currently there is a test in `vmi_lifecycle_test` that abuses the infra placement so the virt-handler will be evicted during VM run. If you have some alternative way to evict or permanently kill virt-handler for a period of time, it will be great because then indeed only the operator tests will touch the KV CR

Reply:
@rmohr PTAL at the new changes. Currently I don't have another way to suggest in VMI lifecycle test, so that we won't that the KV CR there.